src/modules/solution-explorer/solution-explorer-panel/solution-explorer-panel.ts

```diff
@@ -122,7 +122,9 @@ export class SolutionExplorerPanel {
     try {
       await this.solutionExplorerList.openSolution(this.uriOfRemoteSolution);
     } catch (error) {
-      this._notificationService.showNotification(NotificationType.ERROR, `Unable to connect to ProcessEngine on: ${this.uriOfRemoteSolution}`);
+      const genericMessage: string = `Unable to connect to ProcessEngine on: ${this.uriOfRemoteSolution}`;
+      const cause: string = error.message ? error.message : '';
+      this._notificationService.showNotification(NotificationType.ERROR, `${genericMessage}<br />${cause}`);
```

Comment:
I would like to get rid of the spacing
```suggestion
      this._notificationService.showNotification(NotificationType.ERROR, `${genericMessage}<br/>${cause}`);
```

but if you prefer it that way, it's fine with me.

Reply:
Personally, I like the spacing because its easier to see, that the Tag is a standalone Tag.

Reply:
It's easier to see that the day is closed immediately, but that's a matter of opinion.